packages/api/test/functional/technology-order.spec.js

```diff
@@ -53,10 +33,130 @@ const closedOrder = {
 	status: orderStatuses.CLOSED,
 };
 
+test('GET /orders returns all technologyOrder', async ({ client }) => {
+	const { user } = await createUser({ userAppend: { status: 'verified' } });
+	const technology = await Factory.model('App/Models/Technology').create();
+	const technologyOrder = await Factory.model('App/Models/TechnologyOrder').create();
+	await Promise.all([
+		technologyOrder.technology().associate(technology),
+		technologyOrder.user().associate(user),
+	]);
+
+	const response = await client
+		.get('/orders')
+		.loginVia(user, 'jwt')
+		.end();
+	response.assertStatus(200);
+	response.assertJSONSubset([{ ...technologyOrder.toJSON(), technology_id: technology.id }]);
+});
+
+test('GET /orders/:id returns a technologyOrder', async ({ client }) => {
+	const { user } = await createUser({ userAppend: { status: 'verified' } });
+	const technology = await Factory.model('App/Models/Technology').create();
+	const technologyOrder = await Factory.model('App/Models/TechnologyOrder').create();
+	await Promise.all([
+		technologyOrder.technology().associate(technology),
+		technologyOrder.user().associate(user),
+	]);
+
+	const response = await client
+		.get(`/orders/${technologyOrder.id}`)
+		.loginVia(user, 'jwt')
+		.end();
+
+	response.assertStatus(200);
+	response.assertJSONSubset({ ...technologyOrder.toJSON(), technology_id: technology.id });
+});
+
+test('GET /technologies/:id/orders returns all orders for a technology', async ({ client }) => {
+	const { user } = await createUser({ userAppend: { status: 'verified' } });
+	const technology = await Factory.model('App/Models/Technology').create();
+	const technologyOrder = await Factory.model('App/Models/TechnologyOrder').create();
+	await Promise.all([
+		technologyOrder.technology().associate(technology),
+		technologyOrder.user().associate(user),
+	]);
+
+	const response = await client
+		.get(`/technologies/${technology.id}/orders/`)
+		.loginVia(user, 'jwt')
+		.end();
+
+	response.assertStatus(200);
+	response.assertJSONSubset([technologyOrder.toJSON()]);
+});
+
+test('POST /technologies/:id/orders creates a new technologyOrder', async ({ client }) => {
+	const { user } = await createUser({ userAppend: { status: 'verified' } });
+	const technology = await Factory.model('App/Models/Technology').create();
+
+	const response = await client
+		.post(`/technologies/${technology.id}/orders/`)
+		.loginVia(user, 'jwt')
+		.send({
+			quantity: 1,
+			use: 'private',
+			funding: 'has_funding',
+		})
+		.end();
+
+	response.assertStatus(200);
+	response.assertJSONSubset({ user_id: user.id, technology_id: technology.id });
+});
+
+test('PUT orders/:id/update-status technologyOrder status update', async ({ client }) => {
+	const { user } = await createUser({ userAppend: { status: 'verified' } });
```

Comment:
O teste estava passando sem o usuário ser verificado? Pode checar se essa feature está funcionando? 👀👀👀
```suggestion
	const { user } = await createUser({ append: { status: 'verified' } });
```

Reply:
Certo!